### PR TITLE
Cheatsheet for Helm

### PR DESCRIPTION
**Pull request by ImMnan**
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.

Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

This is a comprehensive Cheat sheet, it helpful for all level of users (beginner, intermediate or advance). I have been personally using this for a while now.
Signed-off-by: Manan Patel <immnan333@gmail.com>

### CODE REVIEW

**Comment by ImMnan** [comment]
Do I need to do anything else, I am waiting for this to be approved!

File: content/en/docs/intro/CheatSheet.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+# Cheat Sheet - HELM
+
+Helm cheatsheet featuring  all the necessary commands required to  manage application through Helm.
+
+-----------------------------------------------------------------------------------------------------------------------------------------------
+### Basic interpretations/context
+
+Chart:
+- It is name of your chart in case it has been pulled and untared.
```
Comment (angellk) [inline content/en/docs/intro/CheatSheet.md:9]: minor typo on this line: please change to 'untarred'

File: content/en/docs/intro/CheatSheet.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
+# Cheat Sheet - HELM
+
+Helm cheatsheet featuring  all the necessary commands required to  manage application through Helm.
```
Comment (angellk) [inline content/en/docs/intro/CheatSheet.md:3]: minor change: please change to "manage an application" and remove the extra whitespaces

**Review by angellk — CHANGES_REQUESTED** [review]
Thanks for contributing this cheatsheet @ImMnan ! Once the minor edits are fixed, it can be merged

File: content/en/docs/intro/CheatSheet.md (hunk 1 above)
Comment (ImMnan) [reply content/en/docs/intro/CheatSheet.md:9]: Alright, will do that

**Review by ImMnan — COMMENTED** [review]
Done,

**Review by angellk — APPROVED** [review]
thank you!

**Pull request merged**